components/centraldashboard/public/components/namespace-selector.js

```diff
@@ -196,6 +197,18 @@ export class NamespaceSelector extends PolymerElement {
      * @return {string}
      */
     getDefaultNamespace() {
+        // Restore the user's previous namespace choice
+        const localStorageKey = "selectedNamespace" + 
+            (this.user && "." + this.user || "");
```

Comment:
I think we'll need to change this one as well, in the same spirit as the above comment